docs/Import-reference-genome.md

```diff
@@ -0,0 +1,49 @@
+# Add a new reference genome to the database
+This manual is intended to guide you to load new reference genome(s) to the database. 
+
+### What is Reference Genome (also know as reference assembly)?
+A digital nucleic acid sequence database assembled by scientists as a representative example of a species' set of genes. Reference genomes can be accessed online using 
+dedicated browsers such as [UCSC Genome Browser](https://genome.ucsc.edu/cgi-bin/hgGateway). 
+
+### Why does reference genomes matter?
+Mutation and Segment plots on Patient View are using genomic coordinates. The **cytoband** column in the Mutated Gene table on Study view rely on the Genome Reference Consortium Build 
+matching the version used to analyse the original data (before it was loaded into cBioPortal DB).
```

Comment:
same comment here and elsewhere, I think the rest of the file is okay

Reply:
does not seem you can complete to turn hard wrap off in Intellij but you can give it a big number such as 300.